selfservice/hook/web_hook.go

```diff
@@ -274,29 +274,36 @@ func (e *WebHook) execute(ctx context.Context, data *templateContext) error {
 			attribute.String("webhook.identity.nid", data.Identity.NID.String()),
 		)
 	}
+
 	var (
-		httpClient                       = e.deps.HTTPClient(ctx)
-		async                            = gjson.GetBytes(e.conf, "response.ignore").Bool()
-		parseResponse                    = gjson.GetBytes(e.conf, "can_interrupt").Bool()
-		tracer                           = trace.SpanFromContext(ctx).TracerProvider().Tracer("kratos-webhooks")
-		cancel        context.CancelFunc = func() {}
```

Comment:
The cancel function was not needed IMO. We only used it to cancel the time out, but the underlying HTTP client already has a timeout itself (it's 30 seconds)